### DISCUSSION TITLE
Announcing DreamBerd's £460 pre-seed

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#446] [Announcements]
Hello investors and stake-holders (and community).

Today on this very day we announce our highly successful pre-seed round.

*We're the best!* Please remember that.

## The damage 
 
A whopping 42 of you invested a total of £460.18 in the DreamBerd Foundation. As promised, all your money has gone towards fighting conversion therapy.

*We're against conversion in all its forms, whether that's implicitly interpreting a number as a string, or [making a trans person kill themselves](https://www.mind.org.uk/news-campaigns/news/it-can-completely-destroy-a-person-how-mind-plans-to-make-mps-understand-the-devastating-effects-of-conversion-therapy/).*

As such, we've now donated **£460.18 to the [Stonewall charity](https://www.stonewall.org.uk/our-work/campaigns/ban-conversion-therapy)**. Thank you for making this possible!

For full transparency, we've posted up [full anonymised info](https://github.com/TodePond/DreamBerd/tree/main/docs/investment/pre-seed) about the fundraise. Keep us honest by checking that your investment has been included! Unfortunately the [sweet wrapper deal](https://github.com/TodePond/DreamBerd/releases/tag/v181.81#fn-1) fell through due to logistical issues.

## The moment

Did you miss a chance to invest? Don't worry! Nothing is stopping you from [donating directly to Stonewall](https://donorbox.org/support-stonewall) right now. Or if you prefer, simply find your nearest queer person, and ask for advice on how to help.

If you don't know any queer people then maybe you should.

## The future

The internet is a strange place. It brings out the best and worst in people, myself included. What does it bring out in you?

See you next week,
Lu x

<img width="1094" alt="image" src="https://github.com/TodePond/DreamBerd/assets/15892272/c8e87508-8c16-4f3d-a1b8-6c2387c3f67e">

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v460.18'>Announcing DreamBerd's £460 pre-seed</a>.</em>

**Comment by l3gacyb3ta** [comment]
Woo!!

**Comment by Starpelly** [comment]
UK? ew

**Reply by Magnogen** [reply]
planet earth? ew

**Comment by Cerulity32K** [comment]
based af
